lib/checks/label/label-content-name-mismatch-evaluate.js

```diff
@@ -38,7 +38,8 @@ function curateString(str) {
 }
 
 function labelContentNameMismatchEvaluate(node, options, virtualNode) {
-  const { pixelThreshold, occuranceThreshold } = options || {};
+  const pixelThreshold = options?.pixelThreshold;
+  const occurrenceThreshold = options?.occurrenceThreshold ?? options.occuranceThreshold;
```

Comment:
```suggestion
  const occurrenceThreshold = options?.occurrenceThreshold ?? options?.occuranceThreshold;
```

Reply:
Committed! Sorry about all the simple issues. This is my first GitHub contribution so still have a lot to pick up. Thanks for helping me with it.

Reply:
No problem! Thanks for contributing to axe-core.